packages/manager/modules/vrack/src/dashboard/vrack.module.js

```diff
@@ -43,9 +44,11 @@ angular
     vrackMoveDialog,
   ])
   .component('ovhManagerVrackComponent', component)
+  .service('vrackService', service)
   .config(routing)
   .run(
     /* @ngInject */ ($templateCache) => {
+      $templateCache.put('vrack/partials/delete.html', deletePartials);
```

Comment:
If not possible to use actual moveDialog, create a new component instead of using partials